src/index.html

```diff
@@ -11,7 +11,181 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/ create-application"    
+      method="post"
+      >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form__field">
+            Surname:
+          <input 
```

Comment:
```suggestion
        <label class="form__field">
          Surname:
          <input 
```
Text and input should be on the same level